src/libraries/System.Transactions.Local/src/System/Transactions/Oletx/DtcInterfaces.cs

```diff
@@ -0,0 +1,330 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Reflection;
+using System.Threading;
+using System.Diagnostics.CodeAnalysis;
+using System.Transactions.DtcProxyShim.DTCInterfaces;
+
+namespace System.Transactions.Oletx
+{
+    internal static class NativeMethods
+    {
+        private const int RetryInterval = 50;  // in milliseconds
+        private const int MaxRetryCount = 100;
+
+        // TODO: Is LibraryImport possible here? UnmanagedType.Interface doesn't seem to be supported.
+        // https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ms678898(v=vs.85)
+        [DllImport(Interop.Libraries.Xolehlp, CharSet = CharSet.Unicode)]
```

Comment:
Is [LibraryImport] supported in this scenario? I'm getting errors that UnmanagedType.Interface isn't supported.

Note that this function gets called exactly once in the program's lifetime, so there's no perf aspect here whatsoever.

Reply:
It is supported, but not with `UnmanagedType.Interface`.

Reply:
OK, I guess we should just leave it with [DllImport] then.